vite.config.js

```diff
@@ -0,0 +1,46 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import path from 'path'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue()],
+  build: {
+    lib: {
+      entry: path.resolve(__dirname, 'src/index.js'),
+      name: 'vue-a11y-dialog',
+      fileName: (format) => `vue-a11y-dialog.${format}.js`,
+    },
+    rollupOptions: {
+      // make sure to externalize deps that shouldn't be bundled
+      // into your library
+      external: ['vue'],
+      output: [
+        {
+          format: 'esm',
+          esModule: true,
+          exports: 'named',
+          globals: {
+            vue: 'Vue'
+          }
+        },
+        {
+          format: 'umd',
+          interop: 'esModule',
+          exports: 'named',
+          sourcemap: false,
+          globals: {
+            vue: 'Vue',
+          },
+        },
+        {
+          format: 'cjs',
```

Comment:
I think we should either add the cjs file to package.json exports (see other comment) or, just remove this stanza entirely. I don't have a huge need or opinion for cjs...I just saw that I was making it available in another reference project and sort of went "might as well". So up to you...but I think we should be consistent between export and generation of build.